kalitectl.py

```diff
@@ -219,7 +225,6 @@ def get_pid():
         raise NotRunning(7)  # Unclean shutdown
 
     # TODO: why is the port in django settings!? :) /benjaoming
-    from django.conf import settings
```

Comment:
I'm have problem with settings object cannot find the correct source code. Do you think it has anything to do with this change?  for error detail, please check hipchat